src/OpenTracing/Tag/Tags.cs

```diff
@@ -0,0 +1,139 @@
+
+        /// <summary>
+        ///     PeerHostIpv4 records IPv4 host address of the peer.
+        /// </summary>
+        public static readonly IntOrStringTag PeerHostIpv4 = new IntOrStringTag("peer.ipv4");
+
+        /// <summary>
+        ///     PeerHostIpv6 records the IPv6 host address of the peer.
+        /// </summary>
+        public static readonly StringTag PeerHostIpv6 = new StringTag("peer.ipv6");
+
+        /// <summary>
+        ///     PeerService records the service name of the peer.
+        /// </summary>
+        public static readonly StringTag PeerService = new StringTag("peer.service");
+
+        /// <summary>
+        ///     PeerHostname records the host name of the peer.
+        /// </summary>
+        public static readonly StringTag PeerHostname = new StringTag("peer.hostname");
+
+        /// <summary>
+        ///     PeerPort records the port number of the peer.
+        /// </summary>
+        public static readonly IntTag PeerPort = new IntTag("peer.port");
+
+        /// <summary>
+        ///     SamplingPriority determines the priority of sampling this Span.
+        /// </summary>
+        public static readonly IntTag SamplingPriority = new IntTag("sampling.priority");
+
+        /// <summary>
+        ///     SpanKind hints at the relationship between spans, e.g. client/server.
+        /// </summary>
+        public static readonly StringTag SpanKind = new StringTag("span.kind");
+
+        /// <summary>
+        ///     Component is a low-cardinality identifier of the module, library, or package that is instrumented.
+        /// </summary>
+        public static readonly StringTag Component = new StringTag("component");
+
+        /// <summary>
+        ///     Error indicates whether a Span ended in an error state.
+        /// </summary>
+        public static readonly BooleanTag Error = new BooleanTag("error");
+
+        /// <summary>
+        ///     DbType indicates the type of Database.
+        ///     For any SQL database, "sql". For others, the lower-case database category, e.g. "cassandra", "hbase", or "redis"
+        /// </summary>
+        public static readonly StringTag DbType = new StringTag("db.type");
+
+        /// <summary>
+        ///     DbInstance indicates the instance name of Database.
+        ///     If the jdbc.url="jdbc:mysql://127.0.0.1:3306/customers", instance name is "customers".
+        /// </summary>
+        public static readonly StringTag DbInstance = new StringTag("db.instance");
+
+        /// <summary>
+        ///     DbUser indicates the user name of Database, e.g. "readonly_user" or "reporting_user"
+        /// </summary>
+        public static readonly StringTag DbUser = new StringTag("db.user");
+
+        /// <summary>
+        ///     DbStatement records a database statement for the given database type.
+        ///     For db.type="SQL", "SELECT /// FROM wuser_table". For db.type="redis", "SET mykey "WuValue".
```

Comment:
the Select statement seems broken here (///)

Reply:
resolved